core/src/main/java/org/kiwix/kiwixmobile/core/data/Repository.kt

```diff
@@ -124,14 +124,14 @@ class Repository @Inject internal constructor(
       .subscribeOn(io)
 
   override fun saveNote(noteListItem: NoteListItem): Completable =
-    Completable.fromAction { notesDao.saveNote(noteListItem) }
+    Completable.fromAction { notesRoomDao.saveNote(noteListItem) }
       .subscribeOn(io)
 
   override fun deleteNotes(noteList: List<NoteListItem>) =
-    Completable.fromAction { notesDao.deleteNotes(noteList) }
+    Completable.fromAction { notesRoomDao.deleteNotes(noteList) }
       .subscribeOn(io)
 
   override fun deleteNote(noteUniqueKey: String): Completable =
```

Comment:
Parameter name should be ```noteTitle```

Reply:
Here we are passing noteLists changing to `noteTitle` would be confusing.

Reply:
hi @gouri-panda ,
here too.

Reply:
Here is the `noteUniqueKey` is the `noteTitle`. We are inserting the unique title as `noteUniqueKey`.

Reply:
Sorry i was your last comment.